tensorflow_lite_support/ios/BUILD

```diff
@@ -213,6 +229,7 @@ ios_static_framework(
         ":TFLAudioRecord.h",
         ":TFLAudioTensor.h",
         ":TFLBaseOptions.h",
+        ":TFLExternalFile.h",
```

Comment:
Why do you need to add TFLExternalFile to the Audio framework?

Reply:
[TFLBaseOptions](https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/ios/task/core/sources/TFLBaseOptions.h#L84) also needs `TFLExternalFile` for defining the model file. Before the image searcher APIs were added `TFLExternalFile` was defined in `TFLBaseOptions.h`.
But searcher needed `TFLExternalFile` in `TFLSearchOptions` as well for defining the index file. Hence the class definition  was moved to a separate file. So all frameworks that use `TFLBaseOptions` needs to include `TFLExternalFile.h` as well.

Reply:
I see. Thanks for tha clarification.